libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/DataType.scala

```diff
@@ -69,6 +69,10 @@ object DataType {
     def isSuccess:Boolean = resultCode == 0 || resultCode == 1
   }
 
+  final case class InstallationResult(success: Boolean, code: String, description: String)
+
+  final case class EcuInstallationResult(result: InstallationResult, target: Seq[TargetFilename])
```

Comment:
What would you think about making:
`EcuInstallationResult(success: Boolean, code: String, description: String, target: Seq[TargetFilename])`.
Else when we want to parse the ECU results we'll have to go `DeviceInstallationReport -> ecuResults -> EcuInstallationResult -> code`.
I don't know if it's worth the duplication though.

Reply:
I like having `InstallationResult` class

Reply:
It's just that `EcuInstallationResult` feels so much like a subclass of `InstallationResult` :) But this is more concise, let's go with it.

Reply:
Renamed to `EcuInstallationReport`